tools/defaults.bzl

```diff
@@ -154,14 +154,38 @@ def ng_e2e_test_library(deps = [], tsconfig = None, **kwargs):
         **kwargs
     )
 
-def karma_web_test_suite(deps = [], srcs = [], **kwargs):
+def karma_web_test_suite(name = None, **kwargs):
```

Comment:
Can we make name a required parameter?  Having it as non-required could get weird results

Reply:
done.